default.hbs

```diff
@@ -2,11 +2,20 @@
 <html lang="{{@site.lang}}">
 
 <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Self-closing tags aren't required for void elements according to HTML5 standards.

Reply:
I understand that it is not necessary, according to the standards. It is only necessary to be consistent with the code generated by the ghost platform.

![dawn-code](https://user-images.githubusercontent.com/14000242/103981580-b409c880-5147-11eb-97ce-c078934f596b.png)